src/components/gameRules/gameRules.js

```diff
@@ -0,0 +1,39 @@
+import GAME_MODES from '../../consts/GAME_MODES';
+import './gameRules.css';
+
+const createTextContent = (gameMode, answer) => {
+  return `You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select ${
+    gameMode === GAME_MODES.people ? 'who' : 'what'
+  } from Star Wars is showed on the left (${answer} right now) from available options`;
+};
+
+const createGameRulesElement = (gameMode, answer) => {
+  const containerEl = document.createElement('div');
+  containerEl.classList.add('mode-rules_wrapper');
+
+  const titleEl = document.createElement('div');
+  titleEl.classList.add('mode-rules_title');
+
+  const icon = new Image();
+  icon.src = '../../static/assets/ui/icons/school_24px.png';
+  icon.classList.add('mode-rules_icon');
+  titleEl.appendChild(icon);
+
+  const spanTitleEl = document.createElement('span');
+  const titleText = 'Mode rules';
+  spanTitleEl.innerText = titleText;
+  titleEl.appendChild(spanTitleEl);
+
+  const rulesEl = document.createElement('div');
+  rulesEl.classList.add('mode-rules_content');
+
+  const content = createTextContent(gameMode, answer);
+  rulesEl.innerText = content;
+
+  containerEl.appendChild(titleEl);
+  containerEl.appendChild(rulesEl);
+
+  return containerEl;
+};
+
+export default createGameRulesElement;
```

Comment:
please avoid default exports

Reply:
Why avoid default exports?

Reply:
mainly because other developers across can import this to other module with different name
it cause potential problems with refactoring (e.g.: you are looking usage of createGameRuleElement to change it
's name (due to refactor) and in some place some developer imported it with differnt name - it can occur with potential error on production) 

tldr: its a good practice to avoid potential errors

Reply:
Thank you :)